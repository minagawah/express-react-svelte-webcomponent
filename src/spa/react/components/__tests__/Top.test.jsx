import React from 'react';
import { shallow } from 'enzyme';
import { Top } from '../';

beforeEach(() => {
});

afterEach(() => {
});

it('renders without crashing', () => {
  const wrapper = shallow(<Top />);
  expect(wrapper).toMatchSnapshot();
});
