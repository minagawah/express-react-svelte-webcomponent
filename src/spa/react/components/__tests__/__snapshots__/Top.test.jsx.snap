// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <h1>
    Pizza
  </h1>
  <NavLink
    to="/toppings"
  >
    Toppings
  </NavLink>
  <NavLink
    to="/cloud"
  >
    Cloud
  </NavLink>
</Fragment>
`;
